data/kamailio.cfg

```diff
@@ -353,129 +364,289 @@ onreply_route {
+			jansson_set("string", "short_message", ""+$var(short)+"", "$var(tmp_graylog_json)");
+			jansson_set("string", "full_message", ""+$var(full)+"", "$var(tmp_graylog_json)");
+			jansson_set("string", "ID", "$ci", "$var(tmp_graylog_json)");
+			jansson_set("string", "From", "$fU", "$var(tmp_graylog_json)");
+			jansson_set("string", "To", "$tU", "$var(tmp_graylog_json)");
+			jansson_set("integer", "CallSetupTime", "$var(CS)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "AudioPacketsSent", "$var(PS)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "ExpectedPacketsSent", "$var(ES)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "ComfortNoisePacketsSent", "$var(SPC)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "SilencePacketsSent", "$var(SPS)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "QuietPacketsSent", "$var(QS)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "AudioPacketsReceived", "$var(PR)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "ExpectedPacketsReceived", "$var(ER)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "AudioOctetsReceived", "$var(OR)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "ReceivePacketsLost", "$var(PLR)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "SendPacketsLost", "$var(PLS)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "ReceiveMaxBurstLost", "$var(BL)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "NumLostSequences", "$var(LS)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "ReceiveBurstDuration", "$var(RBDU)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "ReceiveBurstDensity", "$var(RBDE)", "$var(tmp_graylog_json)");
+			jansson_set("string", "SendBurstDuration", "$var(SBDU)", "$var(tmp_graylog_json)");
+			jansson_set("string", "SendBurstDensity", "$var(SBDE)", "$var(tmp_graylog_json)");
+			jansson_set("string", "UsedEncoder", "$var(EN)", "$var(tmp_graylog_json)");
+			jansson_set("string", "UsedDecoder", "$var(DE)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "ReceiveJitter", "$var(JIR)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "SendJitter", "$var(JIS)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "RttMean", "$var(DL_MEAN)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "RttMin", "$var(DL_MIN)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "RttMax", "$var(DL_MAX)", "$var(tmp_graylog_json)");
+			jansson_set("string", "LocalIP", "$var(LIP)", "$var(tmp_graylog_json)");
+			jansson_set("string", "RemoteIP", "$var(RIP)", "$var(tmp_graylog_json)");
+			jansson_set("integer", "Mos", "$var(MOS)", "$var(tmp_graylog_json)");
+
+			$var(graylog_json) = $var(tmp_graylog_json);
+			$var(tmp_graylog_json) = 0;
+
+			if($sht(x=>$ci::callid_count) == $null) $sht(x=>$ci::callid_count) = 0;
+			$sht(x=>$ci::callid_count) = $sht(x=>$ci::callid_count) + 1;
+
+			route(SEND_ES_DATA);
+		}
+	}
+}
+
```

Comment:
Really nice one. I'd like to have the x-rtp stats on elasticsearch too. Could we generalize this part in order to follow the same logic as per the other stats (so valid both for elasticsearch and graylog).